index.html

```diff
@@ -57,28 +69,62 @@ <h2 id="goals">
       Goals
     </h2>
     <p>
-      The goal of the <a href="https://www.w3.org/community/webscreens/">Second
-      Screen Community Group (CG)</a> is to explore use of secondary screens
-      from Web pages. Specifically, the CG provides a venue for discussing and
-      incubating early proposals that aim to:
+      The <a href="https://www.w3.org/community/webscreens/">Second Screen
+      Community Group (CG)</a> explores the use of secondary screens from Web
+      pages. We propose that this Community Group incubate and develop
+      specifications and prototypes of network protocols that implement the
+      <a href="https://w3c.github.io/presentation-api/">Presentation API</a>.
+    </p>
+    <p>
+      This work has three primary purposes:
```

Comment:
It seems odd that Remote Playback API is not mentioned here but is part of the goals below.